capture-sdk/sdk/src/main/java/net/gini/android/capture/view/NavigationBarTopAdapter.kt

```diff
@@ -90,7 +91,7 @@ class DefaultNavigationBarTopAdapter : NavigationBarTopAdapter {
         if (GiniCapture.hasInstance()
             && GiniCapture.getInstance().isBottomNavigationBarEnabled
         ) {
-            viewBinding?.gcNavigationBar?.setOnMenuItemClickListener {
+            viewBinding?.gcNavigationBar?.setOnMenuItemIntervalClickListener {
```

Comment:
[Suggestion] Same as in my previous comment. If the listener we pass in is `IntervalClickListener` then we don't have to use `setOnMenuItemIntervalClickListener`.